### PR TITLE
feat: update feature flags for iota 2.0

### LINKED ISSUE #7870: [Task]: Update feature flags for IOTA 2.0 Testnet
### Task description

Disable everything that we dont need for the public testnet

### PR DESCRIPTION
**Pull request by cpl121**
Closes #7870 

## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/features/wallet.features.ts [hunk 1]
```diff
@@ -18,12 +18,12 @@ const walletFeatures: IWalletFeatures = {
         },
     },
     activityHistory: {
```
Comment (evavirseda) [inline packages/desktop/features/wallet.features.ts:20]: why should the activity history be disabled? :thinking:

Comment (cpl121) [reply packages/desktop/features/wallet.features.ts:20]: I confused `activityHistory` with `addressHistory` :trollface:

File: packages/desktop/features/onboarding.features.ts [hunk 2]
```diff
@@ -65,16 +65,16 @@ const onboardingFeaturesForIotaAlphanet: IOnboardingFeaturesForNetwork = {
         },
     },
     claimRewards: {
-        enabled: false,
-        hidden: true,
+        enabled: true,
```
Comment (evavirseda) [inline packages/desktop/features/onboarding.features.ts:68]: just one more thing :pray: 
I think that `claimRewards` doesn't need to be changed :see_no_evil:

**Review by evavirseda — CHANGES_REQUESTED** [review]

**Review by evavirseda — APPROVED** [review]
looks good :+1:

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**